radis/lbl/loader.py

```diff
@@ -1876,6 +1948,11 @@ def load_and_concat(files, buffer):
                         verbose=verbose,
                         drop_non_numeric=True,
                     )
+                # elif dbformat == "npy":
+                #
+                #     df = npy2df(filename, verbose=verbose)
+                #     # path like : {'v0': path to v0.npy, 'da': path to da.npy}
+
```

Comment:
```suggestion
```
idem

Reply:
Hey @erwanp I think there's some misunderstanding here. I was under the assumption that the `npy2df` function would be called only when the data being loaded is in `npy` format. Since the test example used the cdsd data directly, it must have been loaded using the `cdsd2df` method. 

If you're asking for the general case in case the user wishes to load `npy` data, I am still using the code I had written. Your npy2df file is still there but it was unlinked when I undid your commit. Since the test example wasn't using npy data anyway, I decided to not focus on it for now.